From 4e84a933d036f1b2efbfc67f7b84b9e4dce547b8 Mon Sep 17 00:00:00 2001
From: Vincent Bernat <vincent@bernat.im>
Date: Thu, 9 Jul 2015 00:47:43 +0200
Subject: [PATCH 02/17] spec: use @VERSION@ to put appropriate version

This way, this doesn't need to be updated at each version.
---
 keepalived.spec.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keepalived.spec.in b/keepalived.spec.in
index f8f8fc5..00cf0f2 100644
--- a/keepalived.spec.in
+++ b/keepalived.spec.in
@@ -6,7 +6,7 @@
 
 Summary: HA monitor built upon LVS, VRRP and services poller
 Name: keepalived
-Version: 1.2.19
+Version: @VERSION@
 Release: 5
 License: GPL
 Group: Applications/System
-- 
2.4.6

